frontend/src/pages/register/formValidation.tsx

```diff
@@ -85,13 +88,21 @@ const FormValidation = () => {
       },
     },
   ];
+  const { register: httpRegister, loading } = useContext(AuthContext);
 
-  const submitForm = (/* data: any */) => {
-    // console.log(data);
+  const submitForm = (data: RegisterData) => {
+    httpRegister(
+      data,
+      () => {},
```

Comment:
() => {} ?

Reply:
That's onSucces that currently is not in use, but will be in future.